infrastructure/src/main/java/org/corfudb/infrastructure/NettyServerRouter.java

```diff
@@ -169,6 +180,29 @@ public boolean validateEpoch(CorfuMsg msg, ChannelHandlerContext ctx) {
     }
 
     /**
+     * Validate the clusterId of the CorfuMsg, and send a WRONG_CLUSTER_ID response
+     * if the server clusterId does not match. Ignored if message has ignoreClusterId
+     * flag set to true.
+     *
+     * @param msg   The incoming message to validate.
+     * @param ctx   The context of the channel handler.
+     * @return      True, if the cluster ID matches else false.
+     */
+    public boolean validateClusterId(CorfuMsg msg, ChannelHandlerContext ctx) {
+        UUID clusterId = getClusterId();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Rename "clusterId" which hides the field declared at line 104. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AHiddenFieldCheck)